src/Process/Build.php

```diff
@@ -53,9 +54,11 @@ public function __construct($cwd, $tasks = [])
     /**
      * Starts and returns the running process.
      *
+     * @param callable|null $callback
+     *
      * @return \IteratorAggregate
```

Comment:
Could you fix this one? It doesn't return anything anymore (also maybe you prefer have a return like the last exit code)